jsonschema-hyperschema.xml

```diff
@@ -1722,6 +1722,302 @@ Link: <https://schema.example.com/entry> rel=describedBy
                     Discovering ordered links
                     Multiple self links (for the collection and each item)
                 </cref></t>
+                <t>
+                    In many systems, individual resources are grouped into collections.  Those
+                    collections also often provide a way to create individual item resources with
+                    server-assigned identifiers.
+                </t>
+                <t>
+                    This schema describes a collection where each item representation is
+                    identical to the individual resource item representation, and there
+                    is enough metadata included in the collection representtion to
```

Comment:
`representtion`

Reply:
Thanks- fixed with a rebased commit that only changed this one word.